frontend/assets/javascripts/src/modules/comparisonTable.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * This file:
+ * Controls the listeners for the identity icon menu found in the header
+ * Controls the html document listener to close the identity icon menu
+ * Sets the identity icon returnUrl when a user needs to sign in (controlled via JavaScript for caching reasons)
+ */
+define([
+    '$',
+    'bean'
+], function ($, bean) {
+    'use strict';
+
+    var HOVER_CLASS = 'is-hover';
+    var ACTIVE_CLASS = 'is-active';
+    var COMPARISON_TABLE = document.querySelector('.comparison-table');
+
+    function init() {
+        addHoverListeners();
+        addClickListeners();
+    }
+
+    function addClassToTier(clazz, tier) {
+        var tierElems = COMPARISON_TABLE.querySelectorAll('.comparison-table--clickable[data-tier="'+tier+'"]');
+        for (var i = 0; i < tierElems.length; i++) {
+            tierElems[i].classList.add(clazz);
+        }
+    }
+
+    function addHoverListeners() {
+        $('.comparison-table--clickable').each(function(elem) {
```

Comment:
I might be misreading what this does, but can we not do this with a `:hover` CSS selector and no JS?